pf9/support/commands.py

```diff
@@ -110,7 +110,15 @@ def create(ctx, silent, host, offline, mgmt_plane):
         offline = True
 
     use_localhost = False
-    bundle_exec = 'python /opt/pf9/hostagent/lib/python2.7/site-packages/datagatherer/datagatherer.py'
+    datagatherer_py3 = '/opt/pf9/hostagent/lib/python3.6/site-packages/datagatherer/datagatherer.py'
+    datagatherer_py2 = '/opt/pf9/hostagent/lib/python2.7/site-packages/datagatherer/datagatherer.py'
+    if os.path.isfile(datagatherer_py3):
+        bundle_exec = 'python {}'.format(datagatherer_py3)
```

Comment:
python3 ? to be specific ?

Reply:
This is because the CLI release is not tied to the PMK release. For example, if I push this out today, it will need to work with existing PMK release. And pure py3 support in PMK comes in the next release.